client/landing/gutenboarding/stores/onboard/reducer.ts

```diff
@@ -67,12 +67,24 @@ const pageLayouts: Reducer< string[], OnboardAction > = ( state = [], action ) =
 	return state;
 };
 
+const siteWasCreatedForDomainPurchase: Reducer< boolean, OnboardAction > = (
+	state = false,
+	action
+) => {
+	if ( action.type === 'SET_SITE_WAS_CREATED_FOR_DOMAIN_PURCHASE' ) {
+		return action.siteWasCreatedForDomainPurchase;
+	}
+
```

Comment:
Looks like this reducer needs a path for `RESET_ONBOARD_STORE`.

```suggestion
if ( action.type === 'RESET_ONBOARD_STORE' ) {
	return false;
}
```

Reply:
Fantastic! Thanks for catching that.